flink-state-backends/flink-statebackend-common/src/main/java/org/apache/flink/state/common/PeriodicMaterializationManager.java

```diff
@@ -310,7 +310,7 @@ private void notifyFailureOrCancellation(
                 () ->
                         target.handleMaterializationFailureOrCancellation(
                                 materializationId, upTo, cause),
-                "Task {} materialization:{},upTo:{} failed or canceled.",
+                "Task %s materialization:%d,upTo:%s failed or canceled.",
```

Comment:
nit: add some spaces.
`"Task %s materialization: %d, upTo: %s, failed or canceled."`

Reply:
Thanks for your suggestion @fredia , fixed.